test-suite/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ThreadLocalRandomTest.scala

```diff
@@ -0,0 +1,217 @@
+      checkBounds(317003999, 2002562096)
+      checkBounds(862632362, 1142026599)
+      checkBounds(1427890121, 1219286218)
+      checkBounds(-1574108386, 1636228257)
+      checkBounds(-906455661, -1634427241)
+      checkBounds(-600941210, -1326622990)
+      checkBounds(784503213, -1214499667)
+      checkBounds(1887012585, 966620723)
+      checkBounds(-1028952090, -1629844538)
+      checkBounds(1177745206, 2060996577)
+      checkBounds(-1530572787, 1311494927)
+      checkBounds(-225091256, -201029616)
+      checkBounds(-1624577061, 404594240)
+      checkBounds(582850058, -1481614433)
+      checkBounds(1140369168, -609542932)
+      checkBounds(-1779201251, 2104334764)
+      checkBounds(-922485285, -625675495)
+      checkBounds(464947671, 787431498)
+      checkBounds(640742782, 1992656659)
+      checkBounds(-391198065, -1625837455)
+      checkBounds(1713074993, 2137774205)
+      checkBounds(788987927, 1092726069)
+      checkBounds(-1010524857, 1602499752)
+      checkBounds(-841705591, 838703675)
+      checkBounds(1750248079, 610753575)
+      checkBounds(-1201819578, 698330472)
+      checkBounds(1408484348, -1200755294)
+      checkBounds(1165496379, -1131214886)
+      checkBounds(182115464, 1925130730)
+      checkBounds(1227659366, 49343003)
+      checkBounds(-44588204, 1581213006)
+      checkBounds(-746652264, -1877313645)
+      checkBounds(-1367804909, -236733908)
+      checkBounds(-688797316, 1502002495)
+      checkBounds(1505454505, -621424438)
+      checkBounds(1012590551, 1373499296)
+      checkBounds(742127374, 1999360102)
+      checkBounds(-132299759, -474606603)
+      checkBounds(453028472, -1910125173)
+      checkBounds(1126185715, 1540655275)
+      checkBounds(1684537017, 824396197)
+      checkBounds(-534387535, -1457839852)
+      checkBounds(-252616987, -1445423144)
+      checkBounds(1353546539, -2021734313)
+      checkBounds(93831223, 1735736076)
+      checkBounds(-1952489361, 1322311591)
+      checkBounds(706836020, -1872129716)
+      checkBounds(1876199810, -989606985)
+      checkBounds(1180083473, -1987354544)
+      checkBounds(358830432, -1054448275)
+      checkBounds(-331221423, 1964906328)
+      checkBounds(-692586432, 1473855957)
+      checkBounds(-1850379342, -1891837382)
+      checkBounds(2115982107, 515638616)
+      checkBounds(1250405449, -562976322)
+      checkBounds(1238265711, -1316997587)
+      checkBounds(-174356501, 2506025)
+      checkBounds(1205481279, -1674427890)
+      checkBounds(-217617201, -833593065)
+      checkBounds(20848991, -1440699601)
+      checkBounds(2010553201, 797241229)
+      checkBounds(658643437, 315920491)
+      checkBounds(-1507203912, -507923122)
+
+      expectThrows[IllegalArgumentException](tlr.nextInt(2,1))
+      expectThrows[IllegalArgumentException](tlr.nextInt(1,1))
+    }
+
+    it("should return nextLong under a bound") {
+      val tlr = ThreadLocalRandom.current()
+
+      for (_ <- 1 to 100) {
```

Comment:
Same stuff about a hard-coded list of input data. Also the same for the other tests.